plugins/@grouparoo/mailchimp/src/lib/export/exportProfile.ts

```diff
@@ -34,13 +33,26 @@ export const exportProfile: ExportProfilePluginMethod = async ({
   const mailchimpId = generateMailchimpId(email_address);
   // consider if the the email address has changed
   if (
-    !toDelete &&
     oldProfileProperties["email_address"] &&
     email_address !== oldProfileProperties["email_address"]
   ) {
     const oldMailchimpId = generateMailchimpId(
       oldProfileProperties["email_address"]
     );
+    const oldUserResponse = await getUser(client, listId, oldMailchimpId);
+    const userResponse = await getUser(client, listId, mailchimpId);
+    if (toDelete && !oldUserResponse && !userResponse) {
+      return { success: true };
+    }
+    if (!userResponse && !syncOperations.create) {
```

Comment:
it might not be a create if `oldUserResponse` is found and `userResponse` is not found. (In this case the old user is updated). I think a similar situation could happen with line 51 (checking if it's an update)

I would suggest doing something like `const foundUser = userResponse || oldUserResponse` and then using this new variable to do the comparison.